### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by klorpa**
## About The Pull Request
Fixes various spelling errors.

## Why It's Good For The Game
Looks nicer. Is American or British spelling preferred?

## Changelog
:cl:
spellcheck: fixed a few typos
/:cl:

### CODE REVIEW

**Comment by EvilJackCarver** [comment]
Most of the spelling that the players see are in US English, so things that the player will see should be in US English.

Things like code comments and variables, we don't really care too much about it.

**Comment by cruxxansata** [comment]
American English, yeah. That seems to be the consensus.

File: code/modules/lore_codex/news_data/main.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /datum/lore/codex/category/main_news // The top-level categories for the news thing
 	name = "Index"
 	data = "Below you'll find a list of articles relevant to the current (as of 2562) political climate, especially concerning the Almach Rim \
-	region. Each is labled by date of publication and title. This list is self-updating, and from time to time the publisher will push new \
+	region. Each is labelled by date of publication and title. This list is self-updating, and from time to time the publisher will push new \
```
Comment (EvilJackCarver) [inline code/modules/lore_codex/news_data/main.dm]: EN-US spelling, please

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 2]
```diff
@@ -1502,7 +1502,7 @@ End Citadel Change */
 	color = "#3a6617" // rgb(58, 102, 23)
 
 	glass_name = "Ent's Draught"
-	glass_desc = "You can almost smell the tranquility emanating from this."
+	glass_desc = "You can almost smell the tranquillity emanating from this."
```
Comment (EvilJackCarver) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]: American English spelling, please

**Pull request merged**